shared/src/main/java/uk/gov/di/authentication/shared/entity/AuthenticationType.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.di.authentication.shared.entity;
+
+public enum AuthenticationType {
```

Comment:
Can we rename `AuthenticationType` to `AuthenticationMethod` which IMO seems more accurate to what we are representing with this enum (+ is more inline with the terminology we use for "managing your auth methods".

This will require changes to the AuthenticationAttempts entity field name too